### PR TITLE
[EN/ZH] Update regular stage week 1 result & regular stage week 2 schedule for `GBC 2022 Autumn`

### PR DESCRIPTION
**Pull request by RushFTK**
- Update Regular stage week 1 result.
- Update Regular stage week 2 schedule.
- Clarify rules "teams with the same record".
- ~~Update team roster.~~

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md [hunk 1]
```diff
@@ -173,6 +174,59 @@ The GB Cup 2022 Autumn is run by the Team GB and various community members.
 
 ## Match results
 
+### Reugular stage summary
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:177]:
```suggestion
### Regular stage summary
```

**Review by yobrevelc — DISMISSED** [review]

File: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md [hunk 2]
```diff
@@ -173,6 +174,59 @@ The GB Cup 2022 Autumn is run by the Team GB and various community members.
...
+
+| #[^rs-summary-seed] | Teams | abbr.[^rs-summary-abbr] | MR[^rs-summary-mr] | BR[^rs-summary-br] | BD[^rs-summary-bd] | BU[^rs-summary-bu] | Round 1 | Round 2 | Round 3 | Round 4 |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:179]: also the column numbers don't match. if you need to include the last one, fill it with blank cells.

```suggestion
| #[^rs-summary-seed] | Team | Abbr. | MR[^rs-summary-mr] | BR[^rs-summary-br] | BD[^rs-summary-bd] | BU[^rs-summary-bu] | Round 1 | Round 2 | Round 3 | Round 4 |
```

File: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md [hunk 3]
```diff
@@ -173,6 +174,59 @@ The GB Cup 2022 Autumn is run by the Team GB and various community members.
...
+| 都什么年代还在玩传统音游 | 5 | **6** | **猫猫面馆** | [#1](https://osu.ppy.sh/community/matches/104724218) |
+| 无所谓绿皮龙会出手的 | 1 | **6** | **又一个倒霉蛋** | [#1](https://osu.ppy.sh/community/matches/104725150) |
+| **DAMWONGaming** | **6** | 0 | SV狗都不玩 | [#1](https://osu.ppy.sh/community/matches/104725103) |
+| 要我一直摆烂一直摆烂吗 | 2 | **6** | **均衡教派** | [#1](https://osu.ppy.sh/community/matches/104725874) |
+| EIPSYKONGROO | 4 | **6** | **黎曼猜想** | [#1](https://osu.ppy.sh/community/matches/104726550) |
+
+Sunday, 30 October, 2022:
+
+| Team A |  |  | Team B | Match link |
+| --:  |--: | :-- | :-- | :-: |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:226]:
```suggestion
| --: | :-: | :-: | :-- | :-- |
```

File: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md [hunk 4]
```diff
@@ -173,6 +174,59 @@ The GB Cup 2022 Autumn is run by the Team GB and various community members.
...
+| 22 | 无所谓绿皮龙会出手的 | 绿皮龙 | 0-1 | 1-6 | -5 | 1 | 1-6 倒霉蛋 | 你说的队 |
+| 23 | 俄亥俄州立大学躁狂播放器 |  | 俄亥俄 | 0-1 | 1-6 | -5 | 1 | 1-6 奖励天空 | 石头门 |
+| 24 | SV狗都不玩 | 不玩SV | 0-1 | 0-6 | -6 | 1 | 0-6 DWG | 传统音游 |
+
+### Regular stage week 1
+
+Saturday, 29 October 2022:
+
+| Team A |  |  | Team B | Match link |
+| --: | --: | :-- | :-- | :-: |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:211]: please follow the [template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md?plain=1#L129)

```suggestion
| --: | :-: | :-: | :-- | :-- |
```

File: wiki/Tournaments/GBC/GBC_2022_Autumn/zh.md [hunk 5]
```diff
@@ -173,6 +174,59 @@ GBC 2022 秋季赛由 GB 成员和多位社区成员举办。
...
+| 22 | 无所谓绿皮龙会出手的 | 绿皮龙 | 0-1 | 1-6 | -5 | 1 | 1-6 倒霉蛋 | 你说的队 |
+| 23 | 俄亥俄州立大学躁狂播放器 |  | 俄亥俄 | 0-1 | 1-6 | -5 | 1 | 1-6 奖励天空 | 石头门 |
+| 24 | SV狗都不玩 | 不玩SV | 0-1 | 0-6 | -6 | 1 | 0-6 DWG | 传统音游 |
+
+### 常规赛第 1 周
+
+2022 年 10 月 29 日，星期六：
+
+| A 队 |  |  | B 队 | 比赛链接 |
+| --: | --: | :-- | :-- | :-: |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/zh.md:211]:
```suggestion
| --: | :-: | :-: | :-- | :-- |
```

File: wiki/Tournaments/GBC/GBC_2022_Autumn/zh.md [hunk 6]
```diff
@@ -173,6 +174,59 @@ GBC 2022 秋季赛由 GB 成员和多位社区成员举办。
...
+| 都什么年代还在玩传统音游 | 5 | **6** | **猫猫面馆** | [#1](https://osu.ppy.sh/community/matches/104724218) |
+| 无所谓绿皮龙会出手的 | 1 | **6** | **又一个倒霉蛋** | [#1](https://osu.ppy.sh/community/matches/104725150) |
+| **DAMWONGaming** | **6** | 0 | SV狗都不玩 | [#1](https://osu.ppy.sh/community/matches/104725103) |
+| 要我一直摆烂一直摆烂吗 | 2 | **6** | **均衡教派** | [#1](https://osu.ppy.sh/community/matches/104725874) |
+| EIPSYKONGROO | 4 | **6** | **黎曼猜想** | [#1](https://osu.ppy.sh/community/matches/104726550) |
+
+2022 年 10 月 30 日，星期日：
+
+| A 队 |  |  | B 队 | 比赛链接 |
+| --:  |--: | :-- | :-- | :-: |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/zh.md:226]:
```suggestion
| --: | :-: | :-: | :-- | :-- |
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md (hunk 4 above)
Comment (RushFTK) [reply wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:211]: might template update... so sry just copy last year's format.

**Comment by RushFTK** [comment]
could do rebase if necessary.

File: wiki/Tournaments/GBC/GBC_2022_Autumn/zh.md [hunk 7]
```diff
@@ -104,28 +104,29 @@ GBC 2022 秋季赛由 GB 成员和多位社区成员举办。
 | 23 | **近月少女的礼仪** | ::{ flag=CN }:: **[Zyuuu](https://osu.ppy.sh/users/15389275)**, ::{ flag=CN }:: [tito31158](https://osu.ppy.sh/users/16436669), ::{ flag=CN }:: [Koishi-0514](https://osu.ppy.sh/users/7546593), ::{ flag=CN }:: [yem sky](https://osu.ppy.sh/users/10588209), ::{ flag=CN }:: [FALSEFAST](https://osu.ppy.sh/users/25508274) |
 | 24 | **又一个倒霉蛋** | ::{ flag=CN }:: **[fishbone2445](https://osu.ppy.sh/users/17640814)**, ::{ flag=CN }:: [chana](https://osu.ppy.sh/users/18375016), ::{ flag=CN }:: [ALWW](https://osu.ppy.sh/users/18786195), ::{ flag=CN }:: [KafuChino](https://osu.ppy.sh/users/14826575), ::{ flag=CN }:: [spensll](https://osu.ppy.sh/users/4547744) |
 
-## 赛程: 常规赛第一轮
+## 赛程: 常规赛第 2 周
 
-### 2022 年 10 月 29 日，星期六
+### 2022 年 11 月 5 日，星期六：
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/zh.md:109]:
```suggestion
### 2022 年 11 月 5 日，星期六
```

File: wiki/Tournaments/GBC/GBC_2022_Autumn/zh.md [hunk 8]
```diff
@@ -104,28 +104,29 @@ GBC 2022 秋季赛由 GB 成员和多位社区成员举办。
...
+| EIPSYKONGROO | 俄亥俄州立大学躁狂播放器 | Nov 5 (Sat) 15:30 |
+| 你说的队 | 无所谓绿皮龙会出手的 | Nov 5 (Sat) 17:00 |
+| 番茄炒树莓 | 队名太多不知道用哪个 | Nov 5 (Sat) 19:00 |
+| 都什么年代还在玩传统音游 | SV狗都不玩 | Nov 5 (Sat) 20:00 |
+| 奖励一把大天空 | 黎曼猜想 | Nov 5 (Sat) 20:00 |
+| 上课军训补课反正凑不齐人 | 要我一直摆烂摆烂吗 | Nov 5 (Sat) 20:30 |
+| DAMWONGaming | 猫猫面馆 | Nov 5 (Sat) 20:30 |
+| BMS精灵 | 素鲍小馆丸子面条铺 | Nov 5 (Sat) 22:00 |
+
+### 2022 年 11 月 6 日，星期日：
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/zh.md:122]:
```suggestion
### 2022 年 11 月 6 日，星期日
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**